CHANGES.md

```diff
@@ -4,6 +4,7 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 ### Notable changes
 
+* Updated to [Metronome 0.6.33](https://github.com/dcos/metronome/tree/b8a73dd)
```

Comment:
Usually we add a blank line here @kensipe which helps upon merge conflicts. But I think we can leave this as-is (will do this in an editorial round).